### ISSUE TITLE
Exception in Split Panel setReadOnly

### ISSUE THREAD

**Issue by pmeijer** [webgme/webgme#1402]
It should check if the panel has been instantiated before calling out to it.

https://sentry.io/vanderbilt-university/webgme/issues/274250638/

```
TypeError: Unable to get property 'onReadOnlyChanged' of undefined or null reference
  at self (client/js/Panels/SplitPanel/SplitPanel.js:663)
  at PanelBase (client/js/Panels/SplitPanel/SplitPanel.js:658)
  at this (client/js/PanelBase/PanelBase.js:31)
  at this (client/js/LayoutManager/LayoutManager.js:127)
  at Anonymous function (client/js/WebGME.js:153)
...
(6 additional frame(s) were not displayed)
```

**Comment by pmeijer** [comment]
https://sentry.io/vanderbilt-university/webgme/issues/273841765/

**Comment by pmeijer** [comment]
Closed by #1403
